libr/anal/sign.c

```diff
@@ -480,6 +480,40 @@ static void serialize(RAnal *a, RSignItem *it, char *k, char *v) {
 	}
 }
 
+static RList * deserialize_sign_space(RAnal *a, RSpace *space){
```

Comment:
no space between `*` and `deserialize_sign_space`
```suggestion
static RList *deserialize_sign_space(RAnal *a, RSpace *space){
```